int64 state # 3 == homing signal
float32 goal_x # Sends the goal x
float32 goal_y # Sends the goal y
string message # Send info for each instruction